app/commands/user/update_avatar.rb

```diff
@@ -0,0 +1,12 @@
+class User::UpdateAvatar
+  include Mandate
+
+  initialize_with :user, :avatar
+
+  def call
+    user.update!(
+      avatar:,
+      version: user.version + 1
```

Comment:
I figure if this happens simultaneously with something else enough that the db clashes, then cloudfront won't have got round to caching the new one yet anyway, so I'm not worried about the race.